Documentation/release-notes/csharp-8.md

```diff
@@ -0,0 +1,5 @@
+#### Application and library build and deployment
+
+* [GitHub Issue 5049](https://github.com/xamarin/xamarin-android/issues/5049):
+  *Feature 'XYZ' is not available in C# 7.3. Please use language version 8.0 or greater.*
+  build error could prevent projects from using C# 8.0 language features.
```

Comment:
Release note approved. Thanks!

In the final collated release notes, I'll plan to make one tiny addition to include the version where the behavior started. Feel free to comment back if anything looks bad about this adjustment though.

```markdown
#### Application and library build and deployment

* [GitHub Issue 5049](https://github.com/xamarin/xamarin-android/issues/5049):
  Starting in Visual Studio 2019 version 16.7, build errors similar to *Feature
  'XYZ' is not available in C# 7.3. Please use language version 8.0 or greater.*
  could prevent projects from using C# 8.0 language features.
```